treescript/src/treescript/merge_config.py

```diff
@@ -0,0 +1,99 @@
+"""Configuration for repository merges."""
+
+NEW_ESR_BRANCH = "esr68"
+
+merge_configs = {
```

Comment:
I'm not sure if this config belongs in treescript, or in-tree (particularly as it is also used by `mach try release` to generate the changes required for a staging release.

Reply:
I kept it in-tree because it's then in version control, rather than being in the hook that will call the task. I agree that it would be nicer from the scriptworker's point of view to have this configuration outside, but I'm reluctant to do that without an audit trail of what it's been told to do.

Reply:
There's already a version bump in treescript, would that be sufficient?

Reply:
Possibly, with a bit of work. There is at least the xpcom and weave version number bump that isn't handled, see [here](https://searchfox.org/mozilla-central/source/testing/mozharness/scripts/merge_day/gecko_migration.py#305). I've haven't investigated beyond that.

Reply:
In any case, it was already out of scope for this PR. We'll do it in the next one.

Reply:
> In any case, it was already out of scope for this PR. We'll do it in the next one.
Definitely. It is worth considering if that future change will have impact on how the current one is structured, and I was just asking about it ot ensure that that had been done, and it sounds like it has.